runtime/executor/src/main/java/edu/snu/nemo/runtime/executor/Executor.java

```diff
@@ -161,12 +161,21 @@ public void terminate() {
     @Override
     public void onMessage(final ControlMessage.Message message) {
       switch (message.getType()) {
-      case ScheduleTask:
-        final ControlMessage.ScheduleTaskMsg scheduleTaskMsg = message.getScheduleTaskMsg();
-        final Task task =
-            SerializationUtils.deserialize(scheduleTaskMsg.getTask().toByteArray());
-        onTaskReceived(task);
-        break;
+        case ScheduleTask:
```

Comment:
Indentation seems a bit awkward

Reply:
I think it is okay. Similar indentation: `RuntimeMaster#handleControlMessage`

Reply:
Then can you keep the consistency with the lines below?
`default:` is on a different indentation

Reply:
Yes. I've addressed your comment. Thank you!